internal/command/testdata/backend-unchanged/.terraform/terraform.tfstate

```diff
@@ -1,14 +1,14 @@
 {
     "version": 3,
-    "serial": 0,
+    "serial": 1,
     "lineage": "666f9301-7e65-4b19-ae23-71184bb19b03",
     "backend": {
         "type": "local",
         "config": {
             "path": "local-state.tfstate",
             "workspace_dir": null
         },
-        "hash": 4282859327
+        "hash": 18446744073697443647
```

Comment:
Why is this file committed?

Reply:
Have no idea how this file was changed. Now the PR is clean